esp/esp/program/models/__init__.py

```diff
@@ -1067,18 +1067,28 @@ def getShirtInfo(self):
         if 'class_approved' in teacher_dict:
             query = teacher_dict['class_approved']
             query = query.filter(registrationprofile__most_recent_profile=True)
-            query = query.values_list('registrationprofile__teacher_info__shirt_type',
+            if Tag.getTag('teacherinfo_shirt_type_selection') == 'False':
```

Comment:
Can you use `Tag.getBooleanTag` here?  It will take care of parsing things as `False` or `True` in a slightly more permissive way.